docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>"# Now define a sweep over parameter values, where the two axes are over the# two parameters in the circuit." -&gt; "# Now define a sweep over parameter values, the last axis of dimension 2 is for the parameters 'a' and 'b'"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>